### PR TITLE
Replace remaining instances of `db.<model>` in models with `orm_models`

### PR DESCRIPTION
**Pull request by bunchesofdonald**
Replaces the remaining `db.<model>` references in the models files with `orm_models.<model>`.

### CODE REVIEW

File: src/prefect/server/models/saved_searches.py [hunk 1]
```diff
@@ -30,12 +31,11 @@ async def create_saved_search(
...
 
     Returns:
-        db.SavedSearch: the newly-created or updated SavedSearch
-
+        orm_models.SavedSearch: the newly-created or updated SavedSearch
     """
 
     insert_stmt = (
-        db.insert(db.SavedSearch)
+        db.insert(orm_models.SavedSearch)
```
Comment (abrookins) [inline src/prefect/server/models/saved_searches.py:38]: Why do we still need the database abstraction here -- something about the insert?

**Review by abrookins — COMMENTED** [review]
Looks great! My only concern is that "orm_models" feels like a mouthful, which is entirely subjective. 👍

**Comment by bunchesofdonald** [comment]
> Looks great! My only concern is that "orm_models" feels like a mouthful, which is entirely subjective. 👍

Agreed, `orm_models` is the original name for the file and I just haven't changed that yet.

**Review by abrookins — APPROVED** [review]
LGTM!

**Pull request merged**